osmosis-1/osmosis.zone_assets.json

```diff
@@ -4570,6 +4570,19 @@
       ],
       "_comment": "Pepe Bruce Jenner $PBJ",
       "listing_date_time_utc": "2024-07-12T12:00:00Z"
-    }
+    },
+    {
+      "chain_name": "noble",
+      "base_denom": "ausdy",
+      "path": "transfer/channel-750/ausdy",
+      "osmosis_verified": false,
+      "osmosis_unlisted": true,
```

Comment:
This will add it in preview mode, is this what they want to do? (It used to be a requirement to set as preview until Deposits worked, but that is no longer a requirement)

Reply:
No, I just thought it was still standard practice, will remove